python/pyspark/ml/tests/connect/test_legacy_mode_classification.py

```diff
@@ -19,6 +19,7 @@
 import tempfile
 import unittest
 import numpy as np
+import pandas as pd
```

Comment:
can you import this within `test_binary_classes_logistic_regression` or under `should_test_connect` so the tests can be skipped fine when `pandas` is not installed?

Reply:
isn't pandas always installed in CI image ?